src/dbos-runtime/migrate.ts

```diff
@@ -12,9 +13,8 @@ import {
 } from '../user_database';
 import { db_wizard } from './db_wizard';
 
-export async function migrate(configFile: ConfigFile, logger: GlobalLogger) {
-  let poolConfig: PoolConfig = constructPoolConfig(configFile);
-  poolConfig = await db_wizard(poolConfig);
+export async function migrate(config: DBOSConfigInternal, configFile: ConfigFile, logger: GlobalLogger) {
+  const poolConfig = await db_wizard(config.poolConfig);
```

Comment:
Would the wizard already have been called at this point?  Have you tested it with migrate?